AppInspector/rules/default/cryptography/hash_algorithm.json

```diff
@@ -105,30 +83,23 @@
     "name": "Cryptography: Hash Algorithm (General)",
     "id": "AI009200",
     "description": "Cryptography: Hash Algorithm (General)",
-    "tags": [
-      "Cryptography.HashAlgorithm.General"
-    ],
+    "tags":[ "Cryptography.HashAlgorithm.General" ],
     "severity": "moderate",
     "patterns": [
       {
         "pattern": "hash",
         "type": "string",
-        "scopes": [
-          "code",
-          "comment"
-        ],
+        "scopes": [ "code", "comment" ],
         "modifiers": [ "i" ],
         "confidence": "medium"
```

Comment:
I think we should make this low confidence, 'hash' means other things than crypto.